simply/market.py

```diff
@@ -32,8 +32,10 @@ def __init__(self, time, network=None, grid_fee_matrix=None):
             self.grid_fee_matrix = network.grid_fee_matrix
         self.EPS = 1e-10
         if self.save_csv:
-            match_header = ("time", "bid_id", "ask_id", "bid_actor", "ask_actor", "bid_cluster",
-                            "ask_cluster", "energy", "price")
+            match_header = ["time", "bid_id", "ask_id", "bid_actor", "ask_actor", "bid_cluster",
+                            "ask_cluster", "energy", "price"]
+            if len(self.grid_fee_matrix) > 1:
+                match_header.append('grid_fee')
```

Comment:
could be added in all cases analog to the cluster ids. Values can then be 0 or None

Reply:
maybe rename to included_grid_fee